models/Asset/Video/Thumbnail/Processor.php

```diff
@@ -249,7 +249,7 @@ public static function execute(string $processId): void
                             unlink($steam);
 
                             // set proper permissions
-                            @chmod($storagePath, File::getDefaultMode());
+                            @chmod($storagePath, 0664);
```

Comment:
should we use `0664` or better the Symfony default for creating files? that would be `0666`?

see here: https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Filesystem/Filesystem.php#L658

Reply:
I think this needs to go anyway ... since we're using Flysystem here ...

Reply:
I think `unlink($steam);` should go as well.